packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -26,15 +27,62 @@ Customize error pages as part of the sign-in flow.
 
 ## About error page customization
 
-When using an Okta-hosted flow, you can create a unique sign-in experience by providing a [custom domain](/docs/guides/custom-url-domain/) and creating a [Widget style](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget) that matches your application's look and feel.
+When using redirect authentication, you can create a unique sign-in experience. First, set up a [custom domain](/docs/guides/custom-url-domain/) and then create a [Widget style](/docs/guides/custom-widget/main/#style-the-okta-hosted-sign-in-widget) that matches your application's look and feel.
 
-However, if an error occurs during sign-in, Okta may need to display an error page to the user. To provide a seamless user experience, you can also customize the error page by using the code editor.
+However, if an error happens during sign-in, Okta needs to display an error page to the user. You can also customize the error page by using the code editor.
```

Comment:
I would omit 'However,'.:

"If an error occurs during sign-in, Okta displays an error page to the user. You can use the code editor to customize the error page."

- occurs is better than 'happens', imo
- we can omit needs; Okta is displaying the error page; there's no 'need' about it
- acrolinx will complain about 'by using the code editor', so swapping the order makes it more active voice

Reply:
Done.